ui/apps/platform/src/Containers/NetworkGraph/components/DisplayOptionsSelect.tsx

```diff
@@ -63,6 +78,37 @@ function DisplayOptionsSelect({
                     Port and protocol label
                 </SelectOption>
             </SelectGroup>
+            <SelectGroup label="Selection indicators" key="selection-indicator">
+                <SelectOption key={2} value="selectionIndicator">
+                    <Split>
+                        <SplitItem className="pf-u-mr-xs">
+                            <FilteredEntity width="24px" height="24px" />
+                        </SplitItem>
+                        <SplitItem>Filtered</SplitItem>
+                        <SplitItem className="pf-u-mx-sm">&</SplitItem>
+                        <SplitItem className="pf-u-mr-xs">
+                            <RelatedEntity width="18px" height="18px" />
+                        </SplitItem>
+                        <SplitItem>Related entities</SplitItem>
+                    </Split>
+                </SelectOption>
+            </SelectGroup>
+            <SelectGroup label="Object type labels" key="object-type-labels">
+                <SelectOption key={2} value="objectTypeLabel">
+                    <Split>
+                        <SplitItem className="pf-u-mr-xs">
+                            <NamespaceIcon />
+                        </SplitItem>
+                        <SplitItem className="pf-u-mr-xs">
+                            <DeploymentIcon />
+                        </SplitItem>
+                        <SplitItem className="pf-u-mr-xs">
+                            <CidrBlockIcon />
+                        </SplitItem>
+                        <SplitItem>Labels</SplitItem>
```

Comment:
I think we need an accessibly label for this option, otherwise a screen-reader would read it as "NS D CB Labels"

Reply:
I tried adding a `screenReaderText` for each since they are using the `Badge` component but the screen reader still says it is a bit odd. It says `NS namespace D deployment CB cidr block labels`. I don't know if that really helps, but at least it's better than what I had before. @pedrottimark I'm wondering if you have any suggestions about a case like this